dhall-openapi/openapi-to-dhall/Main.hs

```diff
@@ -174,6 +176,18 @@ parseImport :: String -> Types.Expr -> Dhall.Parser.Parser Dhall.Import
 parseImport _ (Dhall.Note _ (Dhall.Embed l)) = pure l
 parseImport prefix e = fail $ "Expected a Dhall import for " <> prefix <> " not:\n" <> show e
 
+parseNatIntExceptions :: Options.Applicative.ReadM [(String, String)]
+parseNatIntExceptions =
+  Options.Applicative.eitherReader $ \s ->
+    bimap errorBundlePretty id $ result (pack s)
+  where
+    parser = do
+      typ <- some (alphaNumChar)
+      char '.'
+      fld <- some (alphaNumChar)
```

Comment:
Minor suggestion: the parentheses are not necessary

```suggestion
      typ <- some alphaNumChar
      char '.'
      fld <- some alphaNumChar
```

Reply:
Done. Interestingly haskell-language-server does not pick that up.